openprescribing/openprescribing/settings/production.py

```diff
@@ -121,3 +121,5 @@
 
 # Path to import log for pipeline data
 PIPELINE_IMPORT_LOG_PATH = '/home/hello/openprescribing-data/log.json'
+
```

Comment:
Could do with a short comment explaining it's a working directory for the import process, requires several GB of space